bower.json

```diff
@@ -99,6 +99,13 @@
       "libphonenumber-api",
       "momentjs",
       "native-client"
+    ],
+    "lib": [
+      "jquery",
```

Comment:
used for ajax stuff for api.js...should probably be removed eventually, but thats for later.